app/src/main/java/com/mopub/nativeads/InneractiveBannerCustomEvent.java

```diff
@@ -0,0 +1,250 @@
+package com.mopub.nativeads;
+
+import android.content.Context;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.ViewGroup;
+import android.widget.RelativeLayout;
+import com.fyber.inneractive.sdk.external.InneractiveAdRequest;
+import com.fyber.inneractive.sdk.external.InneractiveAdSpot;
+import com.fyber.inneractive.sdk.external.InneractiveAdSpotManager;
+import com.fyber.inneractive.sdk.external.InneractiveAdViewEventsListener;
+import com.fyber.inneractive.sdk.external.InneractiveAdViewUnitController;
+import com.fyber.inneractive.sdk.external.InneractiveErrorCode;
+import com.fyber.inneractive.sdk.external.InneractiveMediationDefs;
+import com.fyber.inneractive.sdk.external.InneractiveMediationName;
+import com.fyber.inneractive.sdk.external.InneractiveUnitController.AdDisplayError;
+import com.fyber.inneractive.sdk.external.InneractiveUserConfig;
+import com.fyber.inneractive.sdk.external.InneractiveUserConfig.Gender;
+import com.mopub.mobileads.CustomEventBanner;
+import com.mopub.mobileads.MoPubErrorCode;
+import java.util.Map;
+
+/**
+ * Inneractive Mopub banner plugin class
+ */
+public class InneractiveBannerCustomEvent extends CustomEventBanner {
+
+  // Set your Inneractive's spot ID!
+  private static String SAMPLE_BANNER_SPOT_ID = "Set_Your_Inneractive_Spot_Id";
```

Comment:
You didn't put our id here.
Also, I don't know if you just followed the documentation and added the other code samples that they were showing afterwards (example make a request, etc), but maybe it would be easier and safer to just download their sdk e copy paste the adapters.

This would apply for both Banner and Interstitial.